Does not work on version 1.8.9

```
---- Minecraft Crash Report ----
// Shall we play a game?

Time: 06.07.21 17:52
Description: Initializing game

java.lang.ExceptionInInitializerError
	at ave.ar(SourceFile:560)
	at ave.am(SourceFile:361)
	at ave.a(SourceFile:310)
	at net.minecraft.client.main.Main.main(SourceFile:124)
Caused by: java.lang.ArrayIndexOutOfBoundsException: 0
	at org.lwjgl.opengl.LinuxDisplay.getAvailableDisplayModes(LinuxDisplay.java:951)
	at org.lwjgl.opengl.LinuxDisplay.init(LinuxDisplay.java:738)
	at org.lwjgl.opengl.Display.<clinit>(Display.java:138)
	... 4 more


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Stacktrace:
	at ave.ar(SourceFile:560)
	at ave.am(SourceFile:361)

-- Initialization --
Details:
Stacktrace:
	at ave.a(SourceFile:310)
	at net.minecraft.client.main.Main.main(SourceFile:124)

-- System Details --
Details:
	Minecraft Version: 1.8.9
	Operating System: Linux (amd64) version 5.12.14-1-MANJARO
	Java Version: 1.8.0_242, AdoptOpenJDK
	Java VM Version: OpenJDK 64-Bit Server VM (mixed mode), AdoptOpenJDK
	Memory: 269362288 bytes (256 MB) / 324534272 bytes (309 MB) up to 3817865216 bytes (3641 MB)
	JVM Flags: 3 total; -Xmx4096m -Xms4096m -Xms256m
	IntCache: cache: 0, tcache: 0, allocated: 0, tallocated: 0
	Launched Version: 1.8.9
	LWJGL: 2.9.4
	OpenGL: ~~ERROR~~ RuntimeException: No OpenGL context found in the current thread.
	GL Caps: 
	Using VBOs: No
	Is Modded: Probably not. Jar signature remains and client brand is untouched.
	Type: Client (map_client.txt)
	Resource Packs: 
	Current Language: ~~ERROR~~ NullPointerException: null
	Profiler Position: N/A (disabled)
	CPU: <unknown>
```

--- a/io.gdevs.GDLauncher.yml
+++ b/io.gdevs.GDLauncher.yml
@@ -53,3 +53,8 @@ modules:
         commands:
           - zypak-wrapper /app/bin/gdlauncher/AppRun
         dest-filename: gd.sh
+  - name: xrandr # if there is a better way to do this please lmk
+    sources:
+    - sha256: 7b99edb7970a1365eaf5bcaf552144e4dfc3ccf510c4abc08569849929fb366e
+      type: archive
+      url: https://www.x.org/archive/individual/app/xrandr-1.5.1.tar.gz

